### PR TITLE
BUG-98856 configurable connector jar Url on RDS

### PR DESCRIPTION
**Pull request by topolyai5**

### CODE REVIEW

File: core/src/main/java/com/sequenceiq/cloudbreak/core/bootstrap/service/host/ClusterHostServiceRunner.java [hunk 1]
```diff
@@ -357,4 +355,15 @@ public String changePrimaryGateway(Stack stack) throws CloudbreakException {
             throw new CloudbreakException("Primary gateway change is not possible because there is no available node for the action");
         }
     }
+
+    private void decoratePillarWithJdbcConnectors(Cluster cluster, Map<String, SaltPillarProperties> servicePillar) {
+        Set<RDSConfig> rdsConfigs = rdsConfigService.findByClusterId(cluster.getOwner(), cluster.getAccount(), cluster.getId());
+        Map<String, Object> connectorJarUrlsByVendor = rdsConfigs.stream()
+                .filter(rds -> StringUtils.isNoneEmpty(rds.getConnectorJarUrl()))
+                .collect(Collectors.toMap(RDSConfig::getDatabaseEngine, RDSConfig::getConnectorJarUrl));
+        Map<String, Object> jdbcConnectors = Collections.singletonMap("jdbc_connectors", connectorJarUrlsByVendor);
```
Comment (mhmxs) [inline core/src/main/java/com/sequenceiq/cloudbreak/core/bootstrap/service/host/ClusterHostServiceRunner.java]: `jdbcConnector` not need for the if statement, so possible to initialize only inside the if

File: orchestrator-salt/src/main/resources/salt/pillar/jdbc/connectors.sls [hunk 2]
```diff
@@ -0,0 +1,2 @@
+jdbc_connectors:
```
Comment (doktoric) [inline orchestrator-salt/src/main/resources/salt/pillar/jdbc/connectors.sls]: is this correct?

Comment (topolyai5) [reply orchestrator-salt/src/main/resources/salt/pillar/jdbc/connectors.sls]: Yes, this is correct

File: core/src/main/java/com/sequenceiq/cloudbreak/converter/RDSConfigRequestToRDSConfigConverter.java [hunk 3]
```diff
@@ -44,6 +45,12 @@ public RDSConfig convert(RDSConfigRequest source) {
         rdsConfig.setCreationDate(new Date().getTime());
         rdsConfig.setStatus(ResourceStatus.USER_MANAGED);
         rdsConfig.setType(source.getType());
+
+        if (!rdsConfig.getDatabaseEngine().equalsIgnoreCase(DatabaseVendor.POSTGRES.name()) && StringUtils.isEmpty(source.getConnectorJarUrl())) {
```
Comment (doktoric) [inline core/src/main/java/com/sequenceiq/cloudbreak/converter/RDSConfigRequestToRDSConfigConverter.java]: if the connector already on the image then ?

**Pull request merged**